deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/src/main/java/org/deeplearning4j/ui/play/PlayUIServer.java

```diff
@@ -102,14 +104,39 @@
     @Parameter(names = {"-f", "--customStatsFile"}, description = "Path to create custom stats file (remote only)", arity = 1)
     private String customStatsFile;
 
+    @Parameter(names = {"-m", "--multiSession"}, description = "Whether to enable multiple separate browser sessions or not", arity = 1)
+    private boolean multiSession;
+
+    StatsStorageLoader statsStorageLoader;
```

Comment:
Intentionally package private? (If so: add comment to that effect). Otherwise private/protected.

Reply:
It was by mistake, I made it private now.